api/internal/target/kusttarget.go

```diff
@@ -423,7 +423,12 @@ func (kt *KustTarget) accumulateResources(
 			if errors.Is(errF, load.ErrHTTP) {
 				return nil, errF
 			}
+			var redErr *load.RedirectionError
+			if errors.As(errF, &redErr) {
+				path = redErr.NewPath
```

Comment:
I think this code is using redirect `Location` path directly.
I am worried that `newPath` will be used by no check.
like the redirected path requires redirecting more, or the redirected path is missing contents.

Reply:
Sorry, Maybe I explained it wrong.

The scenario I was concerned about that the `redErr.NewPath` address returning a `300` status code.

Reply:
I have understood! Sorry ((( I does not implement this case to avoid recursion. If you think that this case should be implemented I'll do it.